src/main/java/org/irmacard/keyshare/common/PinchangeTokenMessage.java

```diff
@@ -0,0 +1,32 @@
+package org.irmacard.keyshare.common;
+
+public class PinchangeTokenMessage extends TokenMessage {
+	private String oldpin;
```

Comment:
In this project it is called `Pinchange`, in `irmago` and `irma_mobile` it is called `Changepin`, can we make this consistent? (Including capitalization, `ChangePin`, see other comment in `irmago`.)